### PR TITLE
Feature/acq 943 add tile for self paced pl

### PR DESCRIPTION
**Pull request by dmantonyuk**
Add tile for Self Paced PL Problem Solving and Computing

## Links
https://codedotorg.atlassian.net/browse/ACQ-944

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Comment by dmantonyuk** [comment]
Online Self-Paced Professional Learning for Teachers

File: pegasus/sites.v3/code.org/public/educate/professional-development-online.haml [hunk 1]
```diff
@@ -198,6 +198,25 @@ theme: responsive_full_width
...
+          %p
+            %strong=hoc_s(:module_label_duration)
+            =hoc_s(:module_duration_2_hrs)
+          %p
+            %strong=hoc_s(:module_label_prerequisites)
+            =hoc_s(:module_prerequisites_none)
+        .content-footer
+          %a.link-button{href: CDO.studio_url("/courses/self-paced-pl-csd-unit1-2023"), aria:{label: hoc_s(:aria_label_call_to_action_teaching_csp)}}
+            =hoc_s(:call_to_action_start_pl)
+          %a.link-button.secondary{href: CDO.studio_url("https://studio.code.org/s/problem-solving-computing"), aria:{label: hoc_s(:aria_label_call_to_action_curriculum_csp)}}
```
Comment (dmcavoy) [inline pegasus/sites.v3/code.org/public/educate/professional-development-online.haml]: I don't think you need the full URL inside the CDO.studio_Url

File: pegasus/sites.v3/code.org/public/educate/professional-development-online.haml [hunk 2]
```diff
@@ -198,6 +198,25 @@ theme: responsive_full_width
...
+            %strong=hoc_s(:module_label_curriculum)
+            =hoc_s(:curriculum_name_problem_solving)
+          %p
+            %strong=hoc_s(:module_label_duration)
+            =hoc_s(:module_duration_2_hrs)
+          %p
+            %strong=hoc_s(:module_label_prerequisites)
+            =hoc_s(:module_prerequisites_none)
+        .content-footer
+          %a.link-button{href: CDO.studio_url("/courses/self-paced-pl-csd-unit1-2023"), aria:{label: hoc_s(:aria_label_call_to_action_teaching_csp)}}
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/educate/professional-development-online.haml]: This needs a new aria-label string that says `Start the Teaching Problem Solving and Computing professional learning program`.

File: pegasus/sites.v3/code.org/public/educate/professional-development-online.haml [hunk 3]
```diff
@@ -198,6 +198,25 @@ theme: responsive_full_width
...
+            =hoc_s(:call_to_action_start_pl)
+          %a.link-button.secondary{href: CDO.studio_url("/s/problem-solving-computing"), aria:{label: hoc_s(:aria_label_call_to_action_curriculum_csp)}}
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/educate/professional-development-online.haml]: This needs a new aria-label string that says `Explore the Teaching Problem Solving and Computing curriculum`.

File: pegasus/sites.v3/code.org/public/images/self-paced-pl-tile-animations.png
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/images/self-paced-pl-tile-animations.png:1]: This doesn't look like it's used anywhere — can it be removed?

Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/images/self-paced-pl-tile-animations.png:1]: This doesn't look like it's used anywhere, can it be deleted?

File: pegasus/sites.v3/code.org/public/images/problem-solving-course-banner.png
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/images/problem-solving-course-banner.png:1]: I updated this image with one that fits better.

**Review by kelbyhawn — COMMENTED** [review]
Left a few comments, once those are addressed I can take another look.

File: pegasus/sites.v3/code.org/public/educate/professional-development-online.haml (hunk 3 above)
Comment (dmcavoy) [reply pegasus/sites.v3/code.org/public/educate/professional-development-online.haml]: Should be "Explore the Problem Solving and Computing curriculum" (no Teaching)

Comment (kelbyhawn) [reply pegasus/sites.v3/code.org/public/educate/professional-development-online.haml]: @dmantonyuk this string needs to be replaced with this one: `aria_label_call_to_action_curriculum_problem_solving`

**Review by kelbyhawn — APPROVED** [review]
Looks great!

**Pull request merged**